app/src/main/java/com/sdpteam/connectout/EventsMapListActivity.java

```diff
@@ -0,0 +1,46 @@
+package com.sdpteam.connectout;
+
+import android.annotation.SuppressLint;
+import android.os.Bundle;
+import android.widget.Switch;
+
+import androidx.lifecycle.ViewModelProvider;
+
+import com.sdpteam.connectout.mapList.MapListModel;
+import com.sdpteam.connectout.mapList.MapListViewModel;
+import com.sdpteam.connectout.mapList.MapListViewModelFactory;
+import com.sdpteam.connectout.mapList.list.ListViewFragment;
+import com.sdpteam.connectout.mapList.map.MapViewFragment;
+
+public class EventsMapListActivity extends WithFragmentActivity {
+
+    private MapViewFragment mapViewFragment;
+    private ListViewFragment listViewFragment;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_events);
+
+        @SuppressLint("UseSwitchCompatOrMaterialCode")
+        Switch mapListSwitch = findViewById(R.id.events_switch);
```

Comment:
The switch is now changed with a toggle button that displays "To Map" when list is displayed and "To List" when map is displayed. This makes it more intuitive since the user sees both.